### PR TITLE
Proper storing / deletion of created zones

### PR DESCRIPTION
**Pull request by Mathu-lmn**

### CODE REVIEW

File: client/main.lua [hunk 1]
```diff
@@ -219,15 +220,22 @@ function createZones()
                     inNoDispatchZone = false
                 end
             })
+            table.insert(nodispatchzones, tmp)
     	end
     end
 end
 
 local function removeZones()
```
Comment (MonkeyWhisper) [inline client/main.lua:232]: Hi, include a nil check to ensure that the zones are not nil when being removed for good practice

**Review by MonkeyWhisper — CHANGES_REQUESTED** [review]

File: client/main.lua (hunk 1 above)
Comment (Mathu-lmn) [reply client/main.lua:232]: Should be good now :)

Comment (MonkeyWhisper) [reply client/main.lua:232]: Thanks so much!

**Comment by MonkeyWhisper** [comment]
Ops, there's a small conflict here. Could you fix it please? 

Thanks!

**Comment by Mathu-lmn** [comment]
Well it looks like someone pushed almost the same feature so mine is useless pretty much

**Comment by MonkeyWhisper** [comment]
> Well it looks like someone pushed almost the same feature so mine is useless pretty much

Sorry about that, I didn't even notice that...

**Pull request closed without merging**